archivebox/core/utils.py

```diff
@@ -1,39 +1,56 @@
-from pathlib import Path
-
 from django.utils.html import format_html
 
-from core.models import Snapshot
+from core.models import Snapshot, EXTRACTORS
+from pathlib import Path
 
 
 def get_icons(snapshot: Snapshot) -> str:
+    archive_results = snapshot.archiveresult_set.filter(status="succeeded")
     link = snapshot.as_link()
     canon = link.canonical_outputs()
-    out_dir = Path(link.link_dir)
+    output = ""
+    output_template = '<a href="/{}/{}" class="exists-True" title="{}">{} </a>'
```

Comment:
I'd like icons to be present for all of the extractors, and greyed out for the ones that are not present. So this should still be `exists-True`/`exists-False` depending on whether or not the successful output is present.

Reply:
@pirate please check if something like what I just pushed works for what you have in mind.